src/components/common/NavMenu.tsx

```diff
@@ -1,14 +1,24 @@
 import React, { useCallback } from 'react';
-import { DatabaseOutlined, ImportOutlined, PlusOutlined } from '@ant-design/icons';
+import {
+  DatabaseOutlined,
+  ImportOutlined,
+  PlusOutlined,
+  SettingOutlined,
+} from '@ant-design/icons';
 import styled from '@emotion/styled';
 import { Menu, MenuProps } from 'antd';
 import { usePrefixedTranslation } from 'hooks';
 import { useStoreActions } from 'store';
-import { NETWORK_IMPORT, NETWORK_NEW, NODE_IMAGES } from 'components/routing';
+import {
+  NETWORK_IMPORT,
+  NETWORK_NEW,
+  NODE_IMAGES,
+  NETWORK_SETTING,
+} from 'components/routing';
 
 const Styled = {
   Menu: styled.div`
-    width: 496px;
+    width: 600px;
```

Comment:
I suggest dropping this down to `550px` to reduce some of the white space on the right side of the menu.